arq/worker.py

```diff
@@ -432,7 +442,7 @@ def run(self) -> None:
             logger.info('%6.2fs → %s(%s)%s', (start_ms - enqueue_time_ms) / 1000, ref, s, extra)
             # run repr(result) and extra inside try/except as they can raise exceptions
             try:
-                async with async_timeout.timeout(timeout_s):
+                async with async_timeout.timeout(timeout_s or None):
```

Comment:
why is this required? Is it related to this PR?

Reply:
I used in a task like this:
```python
async def cronjob(ctx, funcname, crontab, run_at_startup=False, *args, **kwargs):
    rds = ctx['redis']
    cj = CronTab(crontab)  # python-crontab string parser
    while True:
        dt = datetime.now()
        nr = cj.next(dt, default_utc=True)
        if nr or run_at_startup:
            run_at_startup = False
            await asyncio.sleep(nr)
        await rds.enqueue_job(funcname, *args, **kwargs)
 ```

and then define crontab in this way:

```python
func(cronjob, timeout=0)
```
If I pass None as timeout I cannot have the desired behaviour because of this, in worker.py:
```python
timeout_s = self.job_timeout_s if function.timeout_s is None else function.timeout_s
```
So I can schedule crontab remotely, start and stop them when needed, without the limit of timeout.

Reply:
please remove this.